exercises/practice/leap/.approaches/date-addition-time/content.md

```diff
@@ -0,0 +1,19 @@
+# `time::Date` addition
+
+```rust
+use time::{Date, Duration, Month};
+
+pub fn is_leap_year(year: u64) -> bool {
+    (Date::from_calendar_date(year as i32, Month::February, 28).unwrap() + Duration::DAY).day()
+        == 29
+}
+```
+
+This approach may be considered a "cheat" for this exercise.
```

Comment:
This might be good content for a caution block:

````suggestion
```exercism/caution
This approach may be considered a "cheat" for this exercise.
```
````